action.yml

```diff
@@ -67,13 +73,16 @@ inputs:
     description: The setting for `RUST_BACKTRACE` (see https://doc.rust-lang.org/std/backtrace/index.html#environment-variables)
     required: false
   nix-installer-tag:
-    description: The tag of `nix-installer` to use
+    description: The tag of `nix-installer` to use (conflicts with `nix-installer-revision`, `nix-installer-branch`, `nix-installer-pr`)
```

Comment:
IMHO "conflicts" means there will be an error, but if my eyes (and brain) don't deceive me, it seems like the script will just choose which it encounters first. Maybe it should be "takes priority over ...." instead.